packages/react-core/src/demos/RTL/examples/PaginatedTable.tsx

```diff
@@ -58,10 +58,18 @@ import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-ico
 import HandPaperIcon from '@patternfly/react-icons/dist/esm/icons/hand-paper-icon';
 import imgAvatar from '@patternfly/react-core/src/components/assets/avatarImg.svg';
 
-export const PaginatedTableAction = () => {
-  const [translation, setTranslation] = React.useState(translationsEn);
-  const [page, setPage] = React.useState(1);
-  const [perPage, setPerPage] = React.useState(10);
+interface Row {
+  name: string;
+  status: string[];
+  location: string[];
+  lastModified: string[];
+  url: string;
+}
+
+export const PaginatedTableAction: React.FunctionComponent = () => {
+  const [translation, setTranslation] = React.useState<string>(translationsEn);
```

Comment:
The translations aren't strings, but objects for which I think you'll need to create an interface.

Reply:
@wise-king-sullyman would below interface male sense for translation ?
```
interface Translate {
  [key: string]: any;
}
```

Reply:
Yep that would work good enough for this at least!